mllib/src/main/scala/org/apache/spark/ml/tree/impl/RandomForest.scala

```diff
@@ -50,6 +50,7 @@ private[ml] object RandomForest extends Logging {
       numTrees: Int,
       featureSubsetStrategy: String,
       seed: Long,
+      numClasses: Option[Int] = None,
```

Comment:
I think I used to use it when we were passing this down to the indiivudal decission trees as well. I'll back it out.